src/drivers/uefi/lv_uefi_display.h

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file lv_uefi_display.h
+ *
+ */
+
+#ifndef __LV_UEFI_DISPLAY_H__
+#define __LV_UEFI_DISPLAY_H__
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "../../display/lv_display.h"
+
+#if LV_USE_UEFI
+
+#include "lv_uefi.h"
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/**********************
+ * GLOBAL PROTOTYPES
+ **********************/
+
+/**
+ * @brief Create a LVGL display object.
+ * @param handle The handle on which an instance of the EFI_GRAPHICS_OUTPUT_PROTOCOL protocol is installed.
+ * @return The created LVGL display object.
+*/
+lv_display_t * lv_uefi_display_create(
+    void * handle);
```

Comment:
We keep these in one line for short functions.  
```suggestion
lv_display_t * lv_uefi_display_create(void * handle);
```